docs/USAGE.md

```diff
@@ -3,26 +3,44 @@
 The Earthfile configuration in this directory facilitates builds of Space ROS from source code.
 The generated container image is based on Ubuntu 22.04 (Jammy)
 
-## Building the Docker Image
+## Prerequisites
 
-The [Earthly](https://earthly.dev/get-earthly) utility is required to build this image.
+The following software is required to build the Space ROS Docker image:
 
-To build the image, run:
+- [Docker](https://docs.docker.com/get-docker/)
+- [Earthly](https://earthly.dev/get-earthly)
+- [Git](https://git-scm.com/downloads)
+
+## Setup
+
+The image is built using the [Earthly](https://earthly.dev/get-earthly) utility.
+First, clone the Space ROS repository:
 
 ```bash
-./build.sh
+git clone https://github.com/space-ros/space-ros.git
+cd space-ros
 ```
+Make sure docker is running and the user has the necessary permissions to run docker commands.
 
-The build process will take about 20 or 30 minutes, depending on the host computer.
+Space ROS comes in different flavors, the following image variants are available:
 
-The build will use the local [ros2.repos](../ros2.repos) file, and will pick up any local modifications.
-The ros2 repos file can be regenerated or updated using the build target,
+ - `core-image`: The core image contains the ROS 2 core packages, including the ROS 2 client libraries, the ROS 2 command line tools, and the ROS 2 middleware implementations.
+ - `dev-image`: The dev image contains the core image and additional tools for development, such as the ROS 2 build tools, the ROS 2 test tools, and the ROS 2 launch tools.
 
-```
-earthly build +repos-file
+Build the Space ROS Docker image by running the following command:
+
+```bash
+# To build all image variants, use the following command:
+earthly +all --SKIP_BUILD_TEST=true --VCS_REF="$(git rev-parse HEAD)"
+
+# To build a specific image variant, use the following command:
+earthly +core-image --SKIP_BUILD_TEST=true --VCS_REF="$(git rev-parse HEAD)"
```

Comment:
I didn't realise the changes to the documentation were still with this PR. Is this okay or should be break this to a different issue? I have already opened one at #228

Reply:
Should we still keep the changes in this PR?

Reply:
Ah! No that's fine, if you want to follow in #228 then I will move my comments over there. But for the sake of this PR we should remove the changes from the docs, then? Since the commits have to be updated anyway I think we can just drop those in a rebase... Happy to help with that if you need any assistance.

Reply:
Yes agreed. I will start recommitting the work as discussed dropping the documentation part.

Reply:
My bad. I didn't realise the commits were still here. Sorry for that.